main.py

```diff
@@ -1,7 +1,35 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+
+    for el in ["Western", "Action", "Dramma"]:
+        Genre.objects.create(name=el)
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for name, surname in actors:
+        Actor.objects.create(
+            first_name=name,
+            last_name=surname
+        )
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
```

Comment:
It's working in this case, but it's better to add filter by first_name as well according to the task 

![image](https://user-images.githubusercontent.com/104218550/224380137-d1ad4364-f8d5-43eb-93e5-d84e7b07410e.png)